TagManager.php

```diff
@@ -40,6 +40,17 @@ class TagManager extends \Piwik\Plugin
 {
     public static $enableAutoContainerCreation = true;
 
+    public function __construct($pluginName = false)
```

Comment:
Tried using `postLoad()` but it didn't work for me

Reply:
@AltamashShaikh can we move this into the `registerEvents` method maybe? That's because TagManager plugin might not be enabled and then we don't need to do this

Reply:
Moved